pkgs/applications/office/gnucash/default.nix

```diff
@@ -32,7 +33,7 @@ stdenv.mkDerivation rec {
   # raw source code doesn't work out of box; fetchFromGitHub not usable
   src = fetchurl {
     # Upstream uploaded a -1 tarball on the same release, remove on next release
-    url = "https://github.com/Gnucash/gnucash/releases/download/${version}/${pname}-${version}-1.tar.bz2";
+    url = "https://github.com/Gnucash/gnucash/releases/download/${version}/gnucash-${version}-1.tar.bz2";
```

Comment:
This pname hunk (and the one at the end serve no purpose in this patch.

Reply:
Not sure if I understood what you meant, do you mean I should revert those changes to use `${pname}` instead?
fwiw was told in an earlier review ([here](https://github.com/NixOS/nixpkgs/pull/260887#discussion_r1358702911)) to avoid parameterizing pname, which made sense to me at the time.

Reply:
Yes, I mean revert.

If you want to make that change, please do it in a separate commit, because it has nothing to do with the documentation changes. (Multiple commits can be in same PR.)

Reply:
Ah, that makes sense. I split out the commits.